apitests/spec/devices_spec.ts

```diff
@@ -205,8 +222,20 @@ describe('devices api', () => {
     });
 
     describe('get device by id', () => {
-        const enabled_device = "SampleDevice001_648";
-        const disabled_device = "D2";
+
+        var enabled_device: string;
+        var disabled_device: string;
+        var device_with_cmdHistory: string;
+
```

Comment:
Naming: these variables should be end if _id prefix. Also use let instead of var. Usage of var should be discouraged in ES6